### PR TITLE
Ashish4497

### PR DESCRIPTION
**Pull request by ashish4497**

### CODE REVIEW

File: assignments/2. variables/index.md [hunk 1]
```diff
@@ -2,34 +2,39 @@
 ```js
 var name = "Mark";
 ```
+<!-- name is a var-->
```
Comment (nnnkit) [inline assignments/2. variables/index.md:6]: name is the name of the variable.

File: assignments/3. type conversion/index.md [hunk 2]
```diff
@@ -1,36 +1,38 @@
 1. 🎖 What's the outpit of the code below
 ```js
-typeof "Joe";
-typeof 4;
-typeof NaN;
-typeof false;
-typeof function () {};
+typeof "Joe"; //string
```
Comment (nnnkit) [inline assignments/3. type conversion/index.md:3]: It should be `"string"`, `"number"` etc.

File: assignments/4. operators/loops.md [hunk 3]
```diff
@@ -10,7 +10,6 @@ const SPENDING_THRESHOLD = 200;
 const TAX_RATE = 0.08;
 const PHONE_PRICE = 99.99;
 const ACCESSORY_PRICE = 9.99;
-
 var bank_balance = 303.91;
 var amount = 0;
 // your code goes here
```
Comment (nnnkit) [inline assignments/4. operators/loops.md:15]: Solve this problem.

**Review by nnnkit — APPROVED** [review]
Make the required changes and push.

File: assignments/3. type conversion/index.md (hunk 2 above)
Comment (ashish4497) [reply assignments/3. type conversion/index.md:3]: resolve

**Pull request merged**